lib/modules/modules.go

```diff
@@ -225,6 +243,7 @@ func (f Features) IGSEnabled() bool {
 	return f.IdentityGovernanceSecurity
 }
 
+// Deprecated: use other flags do determine cluster features istead of relying on isTeam
```

Comment:
I think if you do this now you might break the linters, as there are existing callers. I'd be particularly worried about e/, as there may be callers there that CI here won't spot.

```suggestion
// TODO(mcbattirola): Deprecate IsTeam once it's unused.
```

Reply:
Gotcha, thank you!